lib/workers/pr/index.ts

```diff
@@ -21,14 +23,61 @@ function noLeadingAtSymbol(input: string): string {
   return input.length && input.startsWith('@') ? input.slice(1) : input;
 }
 
-// TODO: fix types
+function assigneesFromConfig(config: RenovateConfig): Promise<string[]> {
+  return Promise.resolve(config.assignees || []);
+}
+
+async function assigneesFromCodeowners(
+  config: RenovateConfig,
+  pr: Pr
+): Promise<string[]> {
+  if (!config.assigneesFromCodeowners) {
+    return [];
+  }
+
+  const codeowners =
+    (await platform.getFile('CODEOWNERS', pr.targetBranch)) ||
+    (await platform.getFile('.github/CODEOWNERS', pr.targetBranch)) ||
+    (await platform.getFile('.gitlab/CODEOWNERS', pr.targetBranch)) ||
+    (await platform.getFile('docs/CODEOWNERS', pr.targetBranch));
```

Comment:
use `readLocal` from `lib/util/fs`

Please move these new functions to a new file like `lib/workers/pr/assignees.ts`

Reply:
> Please move these new functions to a new file like lib/workers/pr/assignees.ts

Done with https://github.com/renovatebot/renovate/pull/6244/commits/80f0161c2d16cabfda70a619ba82db12c763e851

> use readLocal from lib/util/fs

That won't work because I need these files from the target branch of the PR.

Reply:
> Done with 80f0161

Changed the split a bit to simplify testing and support for setting reviewers https://github.com/renovatebot/renovate/pull/6244/commits/1668b07ccfb8bf664d3162f0ada92d11d77aaf6e